metadata-ingestion/src/datahub/ingestion/source/usage/starburst_trino_usage.py

```diff
@@ -190,9 +190,13 @@ def _convert_str_to_datetime(self, v):
     def _get_joined_access_event(self, events):
         joined_access_events = []
         for event_dict in events:
-            event_dict["create_time"] = self._convert_str_to_datetime(
-                event_dict.get("create_time")
-            )
+            if event_dict.get("create_time"):
+                event_dict["create_time"] = self._convert_str_to_datetime(
+                    event_dict["create_time"]
+                )
+            else:
+                logging.info("The create_time parameter is missing. Skipping ....")
+                continue
 
             event_dict["end_time"] = self._convert_str_to_datetime(
                 event_dict.get("end_time")
```

Comment:
still needs error handling around the `TrinoJoinedAccessEvent(**event_dict)` call